charts/k8ssandra/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: k8ssandra
 description: |
   Provisions and configures an instance of the entire K8ssandra stack. This includes Apache Cassandra, Stargate, Reaper, Medusa, Prometheus, and Grafana.
 type: application
-version: 1.0.5-3
+version: 1.0.5-4
```

Comment:
Missed this still, the github PR review shows the old version, but this is now 1.1.0-SNAPSHOT in the master, so updating this to 1.0.5-4 doesn't work.